heltour/tournament/api.py

```diff
@@ -570,13 +571,13 @@ def get_season_games(request):
                                            'teamplayerpairing__team_pairing__white_team',
                                            'teamplayerpairing__team_pairing__black_team'):
             game_id = get_gameid_from_gamelink(p.game_link)
-            if game_id:
+            if game_id or include_unplayed:
                 r = p.get_round()
                 g = {
                     'league': s.league.name,
                     'season': s.name,
                     'round': r.number if r else None,
-                    'game_id': game_id,
+                    'game_id': game_id if game_id else None,
```

Comment:
I am guessing that when `game_id` is "falsey", then it's probably actually set to `None` already, rendering this line redundant.